qdglue/metrics/grid_based_metrics.py

```diff
@@ -0,0 +1,290 @@
+import dataclasses
+from abc import ABC, abstractmethod
+from typing import Dict, Tuple
+
+import gymnasium
+import numpy as np
+
+from qdglue.types import Feature, Fitness
+
+
+@dataclasses.dataclass
+class DiscreteArchiveMetrics:
+    """Holds statistics about an archive.
+
+    Attributes:
+        coverage: Proportion of cells in the archive that have an elite - always
+            in the range $[0,1]$.
```

Comment:
I added the mkdocs arithmatex extension so writing with dollar signs works now. Careful with escaping backslashes though (probably just use a raw string if you need backslashes).

Reply:
Thank you very much!